library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -105,11 +127,12 @@ internal class TransactionPayloadFragment :
 
     private fun populateUI() {
         if (isAdded && transaction != null) {
-            UiLoaderTask(this).execute(Pair(type, transaction!!))
+            uiLoaderTask = UiLoaderTask(this).apply { execute(Pair(type, transaction!!)) }
         }
     }
 
     private fun setBody(headersString: String, bodyString: String?, isPlainText: Boolean, image: Bitmap?) {
+        uiLoaderTask = null
         headers.visibility = if (headersString.isEmpty()) View.GONE else View.VISIBLE
```

Comment:
I see `isVisible()` from KTX being used earlier in this file. Suggest to replace this line with `isVisible()` too for clarity.

Reply:
Is actually `isVisible()` from `MenuItem` and not from KTX so we can't use that method.